src/main/java/org/isf/priceslist/gui/PricesBrowser.java

```diff
@@ -202,17 +203,13 @@ private JPanel getJPanelSelection() {
 
 	private JLabel getJLabelDescription() {
 		if (jLabelDescription == null) {
-			jLabelDescription = new JLabel();
-			jLabelDescription.setText(getTextDescription());
+			jLabelDescription = new JLabel(getTextDescription());
 		}
 		return jLabelDescription;
 	}
 
 	private String getTextDescription() {
-		String desc = listSelected.getDescription().toUpperCase()+
-		  " ("+ //$NON-NLS-1$
-		  listSelected.getCurrency()+")"; //$NON-NLS-1$
-		return desc;
+		return listSelected.getDescription().toUpperCase() + " (" + listSelected.getCurrency() + ')';
```

Comment:
this method could benefit from the use of StringBuilder

Reply:
Done

Reply:
Thanks!